astro/src/content/docs/_shared/_securing_http_requests.mdx

```diff
@@ -1,11 +1,11 @@
 ---
 ---
-### TLS v1.2
+## TLS v1.2
```

Comment:
This breaks the outline of the two other places this is used

```
grep -Rl _securing_http_requests
./extend/events-and-webhooks/securing.mdx
./lifecycle/migrate-users/connectors/generic-connector.mdx
./customize/email-and-messages/generic-messenger.mdx
```

Suggest we keep it at an h3 and add another header in there. I can take a swing at this if you'd like. (We need to add in the webhook signing section anyway.)